README.md

```diff
@@ -101,6 +101,38 @@ Some features are not yet supported, but are on our roadmap. Check [the roadmap]
       </a>
     </td>
   </tr>
+   <tr>
+    <td>@aries-framework/action-menu</td>
+    <td>
+      <a href="https://npmjs.com/package/@aries-framework/action-menu">
+        <img alt="@aries-framework/action-menu version" src="https://img.shields.io/npm/v/@aries-framework/action-menu"/>
+      </a>
+    </td>
+  </tr>
+  <tr>
+    <td>@aries-framework/bbs-signatures</td>
+    <td>
+      <a href="https://npmjs.com/package/@aries-framework/bbs-signatures">
+        <img alt="@aries-framework/bbs-signatures version" src="https://img.shields.io/npm/v/@aries-framework/bbs-signatures"/>
```

Comment:
As it's private for the moment, in the rendered version I see 'invalid' on the badge. Not related to this PR but I'm wondering if it would be good to publish it so users can start testing AFJ with this kind of credentials (I think we already marked JSON-LD and that stuff as 'experimental' from 0.3.0, and this README file also remarks this WIP status)

Reply:
Thanks @genaris, good points. I removed the `bbs-signatures`. I agree that only public packages should be listed.

Reply:
Oh and I agree with publishing the `bbs-signatures` package. It's already usable without PPV2 being fully merged. The sooner people start testing the better IMO.

Reply:
Yeah sure. We should have a good look at it, test it out again in react native, make sure the flaws are documented (it doesn't work in Node 17 + 18). So if someone is willing to perform these steps, we can set it to public. But otherwise it will automatically be published during the next patch release

Reply:
I'm wondering if it's worth it to document these manual react-native tests somewhere. Since we can't test it programmatically, it may be worth to track it somehow...